data_pipelines/settings.py

```diff
@@ -3,7 +3,7 @@
 
 
 class Settings(BaseSettings):
-    _base_data_path: str = "s3://openepi-data/"
+    base_data_path: str = "s3://openepi-data/"
```

Comment:
Maybe just remove the placeholders so that pydantic will give an error if they are not set (which they always should be)

Reply:
Yes good point